### PR TITLE
[1.3.0] Cut Release 

### PR DESCRIPTION
**Pull request by keefertaylor**
## High Level Overview of Change

Cut release for 1.3.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+This release adds support for the new rippled protocol buffers and support for webpacking this library for use in the browser.
+
+### New Protocol Buffers from rippled
+
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring-JS is now capable of using these protocol buffers.  This change is opt-in and non-breaking. 
+
+To switch to these protocol buffers, pass `true` to the `useNewProtocolBuffer` parameter in `XpringClient`'s constructor. The default for this field is false. 
+
+
+### Webpack functionality
```
Comment (Stormtv) [inline CHANGELOG.md]: Change all references to Webpack to Browser

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,11 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 2.0.0 - Jan 30, 2020
```
Comment (Stormtv) [inline CHANGELOG.md]: 1.3.0

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,11 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+This release adds support for the new rippled protocol buffers and adds browser compatibility support. 
+
+### New Protocol Buffers from rippled
+
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring-JS is now capable of using these protocol buffers.  This change is opt-in and non-breaking.
```
Comment (Stormtv) [inline CHANGELOG.md]: two spaces after last sentence but no where else in the changelog do we do two spaces.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "xpring-js",
-  "version": "1.2.5",
+  "version": "1.3.0",
```
Comment (Stormtv) [inline package.json]: run npm i to generate the package-lock to v1.3.0 also

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,11 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+## 1.3.0 - Jan 30, 2020
+
+This release adds support for the new rippled protocol buffers and adds browser compatibility support. 
+
+### New Protocol Buffers from rippled
+
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring-JS is now capable of using these protocol buffers. This change is opt-in and non-breaking. 
+
+To switch to these protocol buffers, pass `true` to the `useNewProtocolBuffer` parameter in `XpringClient`'s constructor. The default for this field is false.
```
Comment (hbergren) [inline CHANGELOG.md]: Is it worth noting that you need to connect to a `rippled` node that has `gRPC` enabled? (That's true, right?)

Can you configure which `rippled` node you talk to in `Xpring-JS`?

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,11 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### New Protocol Buffers from rippled
+
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring-JS is now capable of using these protocol buffers. This change is opt-in and non-breaking. 
+
+To switch to these protocol buffers, pass `true` to the `useNewProtocolBuffer` parameter in `XpringClient`'s constructor. The default for this field is false. 
+
+
+### Browser Compatibility
+
+This library is now compatible with web browsers. Internally, the library makes different network requests when in the browser. Users can force these request types in the node environment by passing `true` for the `forceWeb` parameter in `XpringClient`'s constructor. This boolean parameter is intended for compatibility testing only. This change is opt-in and non-breaking.
```
Comment (hbergren) [inline CHANGELOG.md]: Maybe note that we plan on removing the `forceWeb` parameter at some point in the future. (Once we have a way to test the web stuff using Puppeteer, and not hacking Node).

**Pull request merged**